connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/lifecycle/InboundConnectorRestController.java

```diff
@@ -47,34 +45,44 @@ public List<ActiveInboundConnectorResponse> getActiveInboundConnectors(
       @RequestParam(required = false) String type) {
     var result =
         inboundManager.query(new ActiveInboundConnectorQuery(bpmnProcessId, elementId, type));
-    return result.stream().map(this::mapToResponse).collect(Collectors.toList());
+    return result.stream().map(this::mapToInboundResponse).collect(Collectors.toList());
   }
 
-  private ActiveInboundConnectorResponse mapToResponse(ActiveInboundConnector connector) {
+  @GetMapping("/inbound/{bpmnProcessId}/{elementId}/{tenantId}/logs")
+  public List<Queue<ActivityLog>> getActiveInboundConnectorLogs(
+      @PathVariable String bpmnProcessId,
+      @PathVariable String elementId,
+      @PathVariable String tenantId) {
+    var result =
+        inboundManager.query(new ActiveInboundConnectorQuery(bpmnProcessId, elementId, null));
```

Comment:
I do not have anything against that, my initial thought was to avoid adding more parameters to the ActiveInboundConnectorQuery contructor. Will add the `tenantId`.

Reply:
But we query later on anyhow so why not add it there.